.travis.yml

```diff
@@ -20,6 +20,7 @@ addons:
       packages:
          - gfortran-8
          - gfortran-9
+         - libgfortran5-dbg
```

Comment:
What does this line do?

Reply:
this package contains debug symbols of libgfortran, which I think would help to get clear error messages.